pennylane/ops/qubit.py

```diff
@@ -1625,13 +1627,35 @@ class ControlledQubitUnitary(QubitUnitary):
 
     >>> U = np.array([[ 0.94877869,  0.31594146], [-0.31594146,  0.94877869]])
     >>> qml.ControlledQubitUnitary(U, control_wires=[0, 1], wires=2)
+
+    Typically controlled operations apply a desired gate if the control qubits
+    are all in the state :math:`\vert 1\rangle`. However, there are some situations where
+    it is necessary to apply a gate conditioned on all qubits being in the
+    :math:`\vert 0\rangle` state, or some mix of the two.
+
+    The state on which to control can be changed by passing a value to
+    `control_values` as either a bit string, or an integer (which is converted
+    to binary representation in Big-Endian format).
+
+    For example, if we want to apply a single-qubit unitary to wire ``3``
+    conditioned on three wires where the first is in state ``0``, the second is
+    in state ``1``, and the third in state ``1``, we can write:
+
+    >>> qml.ControlledQubitUnitary(U, control_wires=[0, 1, 2], wires=3, control_values='011')
+
+    Or equivalently,
+
+    >>> qml.ControlledQubitUnitary(U, control_wires=[0, 1, 2], wires=3, control_values=3)
+
+    since ``011`` is 3 in binary.
```

Comment:
The specific use-case I had in mind was lookup tables (like a quantum ROM) where you have integer or binary addresses with some stored contents. Also sometimes just it's more convenient to write an integer if there is a large number of control bits in some random-like sequence of 0 and 1 :smile:

Reply:
Thanks! I think it's fine keeping it like this, especially if it's useful in certain cases, although I'm tend to prefer only having one type of input per argument. It could be a bit confusing to see `control_values=42` in the wild. That, too me, would sound like there are 42 control values rather than what it's meant to be.

Reply:
Yeah I definitely see how it would be confusing (especially if the integer is something like 10, is that meant to be 10, or 2?). Maybe there is a better way to enable the user to input either value - having two separate keyword arguments, `control_bits` and `control_int` would be one option, I suppose.

Reply:
It avoids the confusion above, although now there are two arguments that basically do the same thing. Not sure if it might be better to just have string inputs allowed, and then the extra step to create it from an int would, if needed, would be on the user.

Reply:
> the extra step to create it from an int would

This is something I have to look up every time :sweat_smile: Okay I'm on board, as this will force me to remember.